upgrading/topics/rhsso/intro.adoc

```diff
@@ -41,3 +43,20 @@ installations can also rollback the patch and server configuration.
 A micro release only contains the artifacts that have changed. For example if Red Hat Single Sign-On 7.2.1 contains changes to
 the server and the JavaScript adapter, but not the EAP adapter, only the server and JavaScript adapter are released and require
 updating.
+
+=== Migration from Keycloak
+
+You can migrate to Red Hat Single Sign-On, the supported Red Hat product, from Keycloak, the community project. Some differences exist in the installed software, such as the following:
+
+* Red Hat Single Sign-On uses JBoss EAP in place of WildFly for the built-in application server
+* Some Keycloak features are not included in Red Hat Single Sign-On, such as some of the Service Provider Interfaces (SPIs).
+
+.Prerequisites
+
+* To learn about new features before the upgrade, review the link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html-single/upgrading_guide/index#changes[changes].
+* Verify that you have installed the correct version of Keycloak as a starting point. To migrate to Red Hat Single Sign-On {project_versionDoc}, you need Keycloak {keycloak_upgrade_version}.
```

Comment:
you need Keycloak {keycloak_upgrade_version} or older

Reply:
@stianst  I understood from Vlasta that users would need a 9.0.x version.  See:  https://issues.redhat.com/browse/KEYCLOAK-10373
Are we relaxing this prerequisite to allow earlier versions?